### PR TITLE
Add Manuel Entry endoint

### PR DESCRIPTION
**Pull request by PantherDave**
Add endpoint to add manual entry of car info. The branch is not named correctly.

### CODE REVIEW

File: .env  [hunk 1]
```diff
@@ -1 +0,0 @@
-FIREBASE_SERVICE_ACCOUNT_KEY_FILE=/Users/adriansilva/src/autoseers/autoseers-firebase-adminsdk-8ep7e-bff36269f0.json
```
Comment (adrdev10) [inline .env :1]: Careful, don't push this.

File: src/main/kotlin/example/com/plugins/Routing.kt [hunk 2]
```diff
@@ -217,5 +217,78 @@ fun Application.configureRouting() {
...
+                                    carInfoData?.get("year") ?: ""
+                                }
+                                carInfoRef.set(mapOf(
+                                    "make" to make,
+                                    "mileage" to mileage,
+                                    "model" to model,
+                                    "year" to year,
+                                    "carHealth" to (carInfoData?.get("carHealth") ?: ""),
+                                ))
+                                call.respond(HttpStatusCode.OK, UploadResponse())
```
Comment (adrdev10) [inline src/main/kotlin/example/com/plugins/Routing.kt:283]: You should be returning the `ManualEntryResponse` not the `UploadResponse`

File: src/main/kotlin/example/com/plugins/Routing.kt [hunk 3]
```diff
@@ -217,5 +217,78 @@ fun Application.configureRouting() {
...
+                                }
+                                val year = request.year.ifBlank {
+                                    carInfoData?.get("year") ?: ""
+                                }
+                                carInfoRef.set(mapOf(
+                                    "make" to make,
+                                    "mileage" to mileage,
+                                    "model" to model,
+                                    "year" to year,
+                                    "carHealth" to (carInfoData?.get("carHealth") ?: ""),
```
Comment (adrdev10) [inline src/main/kotlin/example/com/plugins/Routing.kt:281]: Nice!

File: src/main/kotlin/example/com/plugins/Routing.kt [hunk 4]
```diff
@@ -217,5 +217,78 @@ fun Application.configureRouting() {
...
+
+        post("/manualEntry") {
+            val context = this.coroutineContext
+            val request = call.receive<ManualEntryRequest>()
+            val token = request.token
+            verificationTokenService
+                .verifyAndCheckForTokenRevoked(token = token, getFirebaseToken = true)
+                .collectLatest { verificationState ->
+                    when (verificationState) {
+                        is VerificationState.VerificationStateFailure -> {
```
Comment (adrdev10) [inline src/main/kotlin/example/com/plugins/Routing.kt:229]: I feel like most of these checks are always repeated in other endpoints. Would be nice to clean it up a bit in a different PR 😄

**Review by adrdev10 — COMMENTED** [review]
Looks good! left a couple of questions.

File: .env  (hunk 1 above)
Comment (PantherDave) [reply .env :1]: Should this even be here tho? This isn't private info?

**Review by adrdev10 — APPROVED** [review]

**Comment by adrdev10** [comment]
Tested your PR and it works pretty good!

**Pull request merged**